saleor/graphql/discount/mutations/voucher/voucher_code_bulk_delete.py

```diff
@@ -65,10 +65,12 @@ def clean_codes(cls, codes, errors_list):
         return cleaned_ids
 
     @classmethod
-    def post_save_actions(cls, info, vouchers_code_map):
+    def post_save_actions(cls, info, codes_instances):
         manager = get_plugin_manager_promise(info.context).get()
-        for voucher, code in vouchers_code_map.items():
-            cls.call_event(manager.voucher_updated, voucher, code)
+        cls.call_event(
+            manager.voucher_codes_deleted,
+            codes_instances,
+        )
```

Comment:
since this is a bulk action, you can extend `voucher_codes_deleted` plugin methods to accept additional `webhooks` param, to reduce no of queries sim to this:
```python
        webhooks = get_webhooks_for_event(WebhookEventAsyncType.ATTRIBUTE_UPDATED)
        for attribute in attributes:
            cls.call_event(manager.attribute_updated, attribute, webhooks=webhooks)
```

Reply:
@Air-t but there is no loop, only one event will always be sent so I don't think is necessary

Reply:
I see not sure why I mentioned that - maybe because I saw the for loop at the beginning 🤦